cloudera/tests/test_integration.py

```diff
@@ -0,0 +1,67 @@
+import pytest
+
+from datadog_checks.cloudera import ClouderaCheck
+from datadog_checks.cloudera.metrics import METRICS
+
+
+@pytest.mark.integration
+def test_given_bad_url_when_check_runs_then_service_check_critical(
+    aggregator,
+    dd_run_check,
+    cloudera_check,
+):
+    # Given
+    instance = {
+        'api_url': 'bad_url',
+    }
+    check = cloudera_check(instance)
+    # When
+    dd_run_check(check)
+    # Then
+    aggregator.assert_service_check('cloudera.can_connect', ClouderaCheck.CRITICAL)
+
+
+@pytest.mark.usefixtures('dd_environment')
+@pytest.mark.integration
+def test_given_api_v12_endpoint_when_check_runs_then_service_check_ok_and_metrics(
```

Comment:
We need to check this if we are not going to support version 5 (api v12)

Reply:
I will ask in meeting on Monday.